### PR TITLE
Updated the padding for the main resume content.

### PR DESCRIPTION
**Pull request by mrunankpawar**
## Issue that this pull request solves is when the user opts to add the profile picture the name of the user and the contact details have some space in between.

Closes: #127 

## Proposed changes

Brief description of what is fixed or changed
Changed the left-right padding in the _rezume.scss

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Screenshots

Screenshot 2022-04-03 at 8 01 40 PM

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: ReadME.md [hunk 1]
```diff
@@ -34,19 +34,19 @@ Rezume is a free online portal for generating resumes without worrying about the
...
+  - [ ] [Git History](https://marketplace.visualstudio.com/items?itemName=donjayamanne.githistory) helps you to view git log, file history, compare branches or commits inside VS Code.
+  - [ ] [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) integrates ESLint into VS Code. If you are new to ESLint check the [documentation](http://eslint.org/).
+  - [ ] [ES7 React/Redux/GraphQL/React-Native snippets](https://marketplace.visualstudio.com/items?itemName=dsznajder.es7-react-js-snippets) provides you JavaScript and React/Redux snippets in ES7 with Babel plugin features for VS Code.
+  - [ ] [Todo Tree](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree) `(optional)` quickly searches (using ripgrep) your workspace for comment tags like TODO and FIXME, and displays them in a tree view in the explorer pane. Clicking a TODO within the tree will open the file and put the cursor on the line containing the TODO.
+  - [ ] [`vscode-icons`](https://marketplace.visualstudio.com/items?itemName=vscode-icons-team.vscode-icons) `(optional)` is more of a cosmetic thing, which brings icons to the Visual Studio Code editor.
```
Comment (praveenscience) [inline ReadME.md:49]: Please don't include this file.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: src/styles/_rezume.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .rezume-rezume {
   @include box-shadow(5px);
   background-color: #fff;
-  padding: 75px 100px;
+  padding: 75px 75px;
```
Comment (praveenscience) [inline src/styles/_rezume.scss:4]: If it's both the same, just remove the second value.

**Review by praveenscience — CHANGES_REQUESTED** [review]

File: src/styles/_rezume.scss (hunk 2 above)
Comment (mrunankpawar) [reply src/styles/_rezume.scss]: Made the changes

File: ReadME.md (hunk 1 above)
Comment (praveenscience) [reply ReadME.md:49]: Remove this file and you're good to go! 😊

Comment (mrunankpawar) [reply ReadME.md:49]: Can you help me out with how can I remove it from the PR?

Comment (praveenscience) [reply ReadME.md:49]: Just revert the changes, if you can?

**Review by praveenscience — APPROVED** [review]
👍

**Comment by praveenscience** [comment]
Congrats, thanks. 😊

**Comment by mrunankpawar** [comment]
> Congrats, thanks. 😊

Thank you so much, sorry for the inconveniece.

**Comment by praveenscience** [comment]
> > Congrats, thanks. 😊
> 
> Thank you so much, sorry for the inconveniece.

Not a problem, this is how you learn.

**Pull request merged**